subprojects/docs/src/docs/release/notes.md

```diff
@@ -12,7 +12,6 @@ We would like to thank the following community members for their contributions t
 [George Thomas](https://github.com/smoothreggae),
 [Anja Papatola](https://github.com/apalopta),
 [Björn Kautler](https://github.com/Vampire),
-[David Burström](https://github.com/davidburstrom),
```

Comment:
We wanted to mention David for the idea for https://github.com/gradle/gradle/pull/20002.

Reply:
Thanks! The job only adds missing persons but not removes persons. I found this and manually removed it. I asked about this in the email but nobody answers. Now I added the contributor back.

Reply:
👍